src/org/zalando/stups/mint/worker/external/bucket_storage.clj

```diff
@@ -0,0 +1,14 @@
+(ns org.zalando.stups.mint.worker.external.bucket_storage)
+
+(defmacro StorageException
```

Comment:
I assume, it was a macro before (just called `S3Exception`). But that's probably a good point to change it 👍